test/libsolidity/syntaxTests/natspec/invalid/dosctring_named_return_param_mismatch.sol

```diff
@@ -0,0 +1,7 @@
+abstract contract C {
+    /// @param id Some identifier
+    /// @return No value returned
+    function vote(uint id) public virtual returns (uint value);
+}
+// ----
+// DocstringParsingError: Doc tag "@return No value returned" does not contain the name of its return parameter.
```

Comment:
"Doc tag" or "NatSpec tag" ?

Reply:
`Doc tag` seems to follow what's already there: https://github.com/ethereum/solidity/pull/7837/files#diff-95ab21012d4fd81c0919ae95ac8bf6bbR137

Reply:
Can we change both to "Documentation tag"? I would "define" natspec to mean the string templating language (see ratspec). In standard-json we also just call it "userdoc".

Reply:
Shouldn't the first be `Documentation tag error` then?

Reply:
Definitely against "doc tag" as it reads like "dogtag".